client/src/components/ElementActions/PublishAction.js

```diff
@@ -11,10 +12,18 @@ const PublishAction = (MenuComponent) => (props) => {
   const handleClick = (event) => {
     event.stopPropagation();
 
-    const { id, version, actions: { handlePublishBlock } } = props;
+    const { id, title, version, actions: { handlePublishBlock } } = props;
 
     if (handlePublishBlock) {
       handlePublishBlock(id, 'DRAFT', 'LIVE', version);
+      jQuery.noticeAdd({
+        text: i18n._t(
+          'UnpublishAction.SUCCESS_NOTIFICATION',
+          `Published '${title} successfully`
```

Comment:
Missing a single quote at the end of the variable. Also we should make this a placeholder e.g. `Published {title} successfully` then use `i18n.inject()` - that way the translation value is constant rather than dynamic